src/worker/index.ts

```diff
@@ -66,40 +66,28 @@ self.addEventListener('message', (msg: MessageEvent) => {
 
 async function initStages(config: WorkerConfig) {
   const { speechConfig, stages } = config
+  const promises: Promise<SpeechProcessor>[] = []
   for (const stage of stages) {
     switch (stage) {
       case Stage.VadTrigger:
-        try {
-          processors.push(await VadTrigger.create(speechConfig))
-        } catch (e) {
-          dispatch({
-            eventType: EventType.Error,
-            error: e.message
-          })
-        }
+        promises.push(VadTrigger.create(speechConfig))
```

Comment:
In the future, I'd like to make this setup more flexible, so that users can create their own stages and have them automatically initialized by the pipeline. I know my original version didn't do that either; just pointing it out for later.